guineaPigFacts/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+#server
```

Comment:
Generally, configuration files like this should be excluded from version control because it can expose data about your application that could be used to attack it.  No need to do this in this PR, but this would be good to address in a future pull request

Reply:
Opened issue for this. Will do in next PR. 
Issue: https://github.com/KorynLA/GuineaPigFactsBackend/issues/5